src/tests/Dashboard/action/dictionaryAction.test.js

```diff
@@ -175,6 +193,23 @@ describe('Test suite for dictionary actions', () => {
     });
   });
 
+  it('should handle fetch dictionary concepts network error', () => {
+    moxios.wait(() => {
+      const request = moxios.requests.mostRecent();
+      request.reject({
+        status: 599,
+      });
+    });
+    const expectedActions = [
+      { payload: true, type: '[ui] toggle spinner' },
+      { payload: false, type: '[ui] toggle spinner' },
```

Comment:
@collinewait, I think it would be best if you used the IS_FETCHING action type instead of [ui] toggle spinner.

Reply:
@abulojoshua1, both issues have been resolved.